docs/authorization.md

```diff
@@ -230,6 +230,42 @@ i.e. a claimed resource is bound to the same maximal permission policy. Only the
 
 TBD: Example
 
+## Permission Claims Authorizer
+
+The requested resource type might be a claimed resource of a PermissionClaim in an APIExport
+referenced by an APIBinding in the workspace being accessed.
+In this case access to the requested resource is constrained by the field `verbs.restrict` of the underlying permission claim.
+
+If the requested resource type is claimed by different API Exports in the same workspace,
+the intersection of all restrict verbs of all APIExports is being granted.
+
+Valid values for the `verbs.restrict` field follow the same rules as the `verbs` field in PolicyRules:
+
+- It is required.
+- `"*"` matches all verbs. If present, it must be the only entry.
+- The list must not be empty.
+
+If the APIExport of the requested and claimed resource has a maximum permission policy configured,
+access to the requested resource can be further constrained as explained in the "API binding authorizer" section.
+
+Example: Given an APIExport `foo` access to `bar.some.group` has to be restricted to read-only access:
+
+```yaml
+apiVersion: apis.kcp.dev/v1alpha1
+kind: APIExport
+metadata:
+  name: foo
+spec:
+...
+  permissionClaims:
+  - resource: bars
+    group: some.group
+    identityHash: 580de6010242a491e8a072512753badf80887af5d569b9980f661f246b29f804
+    verbs:
+      claimed: ["*"]
```

Comment:
It might be helpful to have a matrix of how `claimed` and `restrict` interact. The example in both this section and the `Permission Claims Authorizer` look the same to me, but I'm not understanding what permissions are actually granted.

Reply:
Indeed! There is one important interaction point. `claimed` verbs are only effective in the virtual apiexport apiserver however `restrict_to` (I like the term) verbs are effective both in the regular workspace authorizers and in virtual apiexport apiserver as well. I will document how they interact here 👍